### DISCUSSION TITLE
set image command in TriggerTemplate

### DISCUSSION THREAD

**Discussion by rupeshsahuoc** [tektoncd/triggers#1138] [Q&A]
Is "kubectl set image" command works in TriggerTemplate?

I got error
error: no kind "TriggerTemplate" is registered for version "triggers.tekton.dev/v1alpha1" in scheme "k8s.io/kubernetes/pkg/kubectl/scheme/scheme.go:28"

**Comment by dibyom** [comment]
hi @rupeshsahuoc sorry for the delay - could you provide more details on what you are trying to accomplish by using `kubectl set image` for a TriggerTemplate?

**Reply by rupeshsahuoc** [reply]
I was trying to change the image of a container using *set image* command. As we are using the same command for stateful sets, demon sets, deployments.

**Reply by dibyom** [reply]
which container in particular? The kubectl set image command is specific to the k8s workloads like deployments, daemon sets etc.
